packages/office-addin-debugging/src/commands.ts

```diff
@@ -75,7 +76,7 @@ export async function start(manifestPath: string, platform: string | undefined,
         }
 
         await startDebugging(manifestPath, appTypeToDebug, app, debuggingMethod, sourceBundleUrlComponents,
-            devServer, devServerPort, packager, packagerHost, packagerPort, enableDebugging, enableLiveReload, openDevTools);
+            devServer, devServerPort, packager, packagerHost, packagerPort, enableDebugging, enableLiveReload, openDevTools, document);
```

Comment:
We should consider a breaking change which uses an object with properties rather than an ever-growing list of parameters on this function. It could provide better flexibility in the future. This isn't required per-se for this change, but something I am thinking about and felt might be good to mention.

The downside is that it would be a breaking change. We could consider adding this as another method, e.g. startDebuggingWithOptions(options: StartDebugginOptions) and then this implementation could call that. It would keep the current function limited at the current functionality, or we could continue to extend it, but add the other method for better code design. We might want to consider separating this because for example, the packager stuff is not really as desired since ReactNative debugging is moving to being direct debugging only.

This is just food for thought.